### DISCUSSION TITLE
rest-api upload to nightscout has failed!!

### DISCUSSION THREAD

**Discussion by YOUSIFALHAJ** [NightscoutFoundation/xDrip#2855] [General]
![IMG_20230526_125311_188](https://github.com/NightscoutFoundation/xDrip/assets/111284375/4a66acfa-1459-4a3f-bd12-021f9523f43d)
I am as follower stop receiving reading this is only today morning also when i try to check reading in nightscout platform directly from the browser also it is not working

**Comment by Navid200** [comment]
Would you please log into Google Cloud and post a screenshot of the status page.
This is what I mean by the status page: https://navid200.github.io/xDrip/docs/Nightscout/Status.html

There is a lot on that page that could answer many questions right away.

**Reply by LuisPuertollano** [reply]
I think I have a similar problem. Any help here please?

![Screenshot_20230601_132426_xDrip+](https://github.com/NightscoutFoundation/xDrip/assets/77344100/d9301364-3206-4846-956a-3c06cdbc2dd5)

**Reply by YOUSIFALHAJ** [reply]
I have updated xdrib then it works perfectly

**Reply by LuisPuertollano** [reply]
Im on last stable version, and still with the same problems. ..are you on stable or on beta branch? I think maybe I have a mongo db full?

**Reply by yoxud** [reply]
Had the same thing happened to me yesterday and my mongodb was not full (300 MB out of 512) . I deleted my active db cluster on atlas and redeployed my nightscout pointing to an old unused cluster I had and all is good now. I did notice my nonworking cluster was version 6.06 and the working one is 6.0.5, dont know if that has something to do with it.
